elasticsearch/ci-env.sh

```diff
@@ -4,14 +4,19 @@ set -o xtrace
 
 PROMETHEUS_EXPORTER_URL=${PROMETHEUS_EXPORTER_URL:-$MAVEN_REPO_URL/org/elasticsearch/plugin/prometheus/prometheus-exporter/$PROMETHEUS_EXPORTER_VER/prometheus-exporter-$PROMETHEUS_EXPORTER_VER.zip}
 OPENDISTRO_URL=${OPENDISTRO_URL:-$MAVEN_REPO_URL/com/amazon/opendistroforelasticsearch/opendistro_security/$OPENDISTRO_VER/opendistro_security-$OPENDISTRO_VER.zip}
+INGEST_PLUGIN_URL=${INGEST_PLUGIN_URL:-$MAVEN_REPO_URL/org/elasticsearch/plugin/ingest/openshift/$INGEST_PLUGIN_VER/openshift_ingest-plugin-$INGEST_PLUGIN_VER.zip}
 
 if [[ "${OPENSHIFT_CI:-}" == "true" ]]; then
     # This flag is set during CI runs. If no ARG was passed in,
     # default to maven.org.
     export ES_ARCHIVE_URL=https://github.com/openshift/origin-aggregated-logging/releases/download/elasticsearch-oss-$ES_VER/elasticsearch-oss-$ES_VER.zip
     export OPENDISTRO_URL=https://github.com/openshift/origin-aggregated-logging/releases/download/opendistro_security-$OPENDISTRO_VER/opendistro_security-$OPENDISTRO_VER.zip
 
+    # TODO[lvlcek]: We need to download official releases from ViaQ repo. Getting it from lukas-vlcek is temporary.
+    INGEST_PLUGIN_VER=$(echo $INGEST_PLUGIN_VER | cut -d'-' -f1)
+    export INGEST_PLUGIN_URL=https://github.com/lukas-vlcek/elasticsearch-openshift-ingest-plugin/releases/download/v$INGEST_PLUGIN_VER/openshift-ingest-plugin-$INGEST_PLUGIN_VER.zip
```

Comment:
Shouldnt we "correct" the upstream url to point to a release published here https://github.com/ViaQ/elasticsearch-openshift-ingest-plugin

Reply:
Ok, I see what you mean @jcantrill
The thing is that we need to decide and control who, how and when cuts new releases for https://github.com/ViaQ/elasticsearch-openshift-ingest-plugin/

At this point it is just manual process. I created some tags in my clone for which I manually built releases and uploaded them to https://github.com/lukas-vlcek/elasticsearch-openshift-ingest-plugin/releases
(^^ This is what @vimalk78 used for his tests btw...)

Now, there is no process that is syncing tags between my clone and official ViaQ upstream.
We can either create tags manually or we can adopt similar approach that is used in prometheus-plugin – which means that whenever there is a commit that includes this property file without `-SNAPSHOT` suffix here ...
https://github.com/vvanholl/elasticsearch-prometheus-exporter/blob/master/gradle.properties#L3

... then a new tag is created automatically and a new release is built and uploaded automatically as well. See https://github.com/vvanholl/elasticsearch-prometheus-exporter/blob/master/.github/workflows/ci.yml#L39

We should decide how we want to control this. I would prefer fully automated approach (i.e. the samy way the prometheus plugin does it), but I am OK if you decide to do it differently.

//cc @ewolinetz

Reply:
BTW, I do not think I have privs to create tags on ViaQ repo (the upstream). Which means if we decide we will create and upload release manually then I believe we need to ask @ewolinetz to do this.